tests/test_bus_finder.py

```diff
@@ -16,21 +24,21 @@ class BusFinderTests(unittest.TestCase):
 
     @patch('os.path')
     @patch('omxplayer.bus_finder.glob')
-    @patch('__builtin__.open', new=MOCK_OPEN)
+    @patch('{}.open'.format(builtin), new=MOCK_OPEN)
```

Comment:
This is duplicated quite a lot, we should probably pull it out to `builtin_open_module`